cdap-docs/admin-manual/source/hadoop-compatibility.rst

```diff
@@ -116,13 +116,13 @@ Hadoop components, as required:
 +----------------+---------------------------------+---------------------------------------+
 | **Spark**      | Apache                          | Versions 1.2.x through 1.6.x          |
 +                +---------------------------------+---------------------------------------+
-|                | |cdh-title|                     | |cdh-versions| |note-4|               |
+|                | |cdh-title|                     | Version 1.6.x through 2.2.x           |
```

Comment:
Isn't this column supposed to have the cdh, hdp, and mapr versions?

Reply:
its title is `Spark` versions which displays distro versions. It seems confusing.

Reply:
There is no point in having one row per distro if it's just the same Spark version. The point of this document is to map what we need to distro versions.

Reply:
I wonder showing distro versions for each hadoop component would be helpful? Vs showing highest versions supported by those distros for each component.

Reply:
The rest of the document shows distro versions. The reason this document exists is because it's hard to know which distro version has which version of which component

Reply:
The version for Apache is the one that should have been changed, and it is different in 4.3 than 5.0.

Reply:
Hmm, I agree. We should change supported version for each hadoop component to corresponding component version rather than distro version, right?

Reply:
No, the usefulness of this doc comes from the fact that the *distro* version is listed. This entire doc is so that somebody can go it and look to see if they can install CDAP on their distro. They don't care about the component version.

Reply:
@albertshau The jira also has fix version for 4.3.5. But it seems the change is only needed for 5.0 docs, right?

Reply:
we should still update the upper bound in this doc, as we support up to spark 2.2.x in 4.3.x